utils/testing.ts

```diff
@@ -36,6 +36,9 @@ export async function compareFormData(formData: Record<string, unknown>, entries
     // Read the file asynchronously.
     const reader = new FileReader();
     reader.onload = function (e) {
+      if (e.target === null) {
+        throw new Error("e.target === null");
+      }
```

Comment:
Why do you need that in the test? Would an error here just break the function, not call the "expect" below and mark that test case false-positive ?